doc/source/ray-air/getting-started.rst

```diff
@@ -3,37 +3,46 @@
 Ray AI Runtime (AIR)
 ====================
 
-Ray AI Runtime (AIR) is an open-source toolkit for building end-to-end ML applications. By leveraging Ray, its distributed compute capabilities, and its library ecosystem, Ray AIR brings scalability and programmability to ML platforms.
+Ray AI Runtime (AIR) is a scalable and unified toolkit for ML applications. AIR enables easy scaling of individual workloads, end-to-end workflows, and popular ecosystem frameworks. Unlike other toolkits, AIR can scale workloads with just Python by leveraging the Ray unified compute framework.
 
-.. tip::
-    **Getting involved with Ray AIR.** Fill out `this short form <https://forms.gle/wCCdbaQDtgErYycT6>`__ to get involved. We'll be holding office hours, development sprints, and other activities as we get closer to the Ray AIR Beta/GA release. Join us!
+.. image:: images/air.png
 
-.. image:: images/ai_runtime.jpg
+AIR comes with ready-to-use libraries for :ref:`Data Prep <datasets>`, :ref:`Training <train-docs>`, :ref:`Tuning <tune-main>`, :ref:`Reinforcement Learning <rllib-index>`, :ref:`Serving <rayserve>`, and :ref:`Batch Scoring <air-predictors>`, as well as an ecosystem of integrations.
 
+Get started by installing Ray AIR:
 
-Ray AIR focuses on two functional aspects:
+.. code:: bash
 
-* It provides scalability by leveraging Ray’s distributed compute layer for ML workloads.
-* It is designed to interoperate with other systems for storage and metadata needs.
+    pip install -U ray[air]
 
-Ray AIR consists of five key components:
+Why Ray AIR?
```

Comment:
IMO this feels really overwhelming to have all this presented to me at once.

Reply:
What's overwhelming? The diagram or the introduction and text?

Reply:
The single python script diagram is hard to read; the fonts a bit small and code sections blurry. I know it's from our summit slides.

Reply:
the text is